database/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP TABLE earthquakes CASCADE;
+DROP TABLE alerts CASCADE;
+DROP TABLE status CASCADE;
+DROP TABLE mag_types CASCADE;
+DROP TABLE networks CASCADE;
+DROP TABLE types CASCADE;
+
+CREATE TABLE alerts (
+    alert_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    alert_value VARCHAR(6) UNIQUE NOT NULL,
+    PRIMARY KEY (alert_id)
+);
+
+CREATE TABLE networks (
+    network_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    network_name VARCHAR(2) UNIQUE NOT NULL,
+    PRIMARY KEY (network_id)
+);
+
+CREATE TABLE types (
+    type_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    type_value VARCHAR UNIQUE NOT NULL,
+    PRIMARY KEY (type_id)
+);
+
+CREATE TABLE status (
+    status_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    status VARCHAR UNIQUE NOT NULL,
+    PRIMARY KEY (status_id)
+);
+
+CREATE TABLE mag_types (
+    mag_type_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    mag_type_value VARCHAR(3) UNIQUE NOT NULL,
+    PRIMARY KEY (mag_type_id)
```

Comment:
I think it'll be better if all occurrences of 'mag_type' is changed to `magtype_...`